src/theme/theme.ts

```diff
@@ -99,6 +99,29 @@ const themeOptions: ThemeOptions = {
         },
       },
     },
+    MuiButtonBase: {
+      defaultProps: {
+        disableRipple: true,
+      },
+    },
+    MuiButton: {
+      styleOverrides: {
+        root: {
+          '&:hover': {
+            boxShadow: 'none',
+          },
+        },
+      },
+    },
+    MuiMenuItem: {
+      styleOverrides: {
+        root: {
+          '&.Mui-selected': {
+            backgroundColor: '#954661', //CollageRed[500]
```

Comment:
Changing the MUI Selected property only changes the select on dropdowns but IMO we want to have that color across the entire UI. That's the advantage of theme colors.
If you change the primary color (`theme > palette > primary > main`) I think you achieve the same result but it also spreads to other components on the UI like buttons and more